app/components/ui/power-select-multiple/before-options.js

```diff
@@ -0,0 +1,20 @@
+// import PowerSelectBeforeOptionsComponent from 'ember-power-select/components/power-select/before-options'
+import PowerSelectBeforeOptionsComponent from '@glimmer/component'
+import { action } from '@ember/object'
+
+export default class UiPowerSelectMultipleBeforeOptionsComponent extends PowerSelectBeforeOptionsComponent {
+  get allSelected () {
+    const { options, selected } = this.args.select
+    return selected && selected.length === options.length
+  }
+
+  @action
+  toggleAll () {
+    const { options, selected } = this.args.select
+    if (this.allSelected) {
+      return this.args.select.actions.select(null)
```

Comment:
I think we're good without `returning` here and just nesting line 18 into an `else`